src/components/organisms/UiList/stories/ListBox.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <UiList
```

Comment:
Can you prepare longer list to show scrollbar and check ii looks like in the design?